public/app/features/dashboard/dashgrid/PanelLinks.tsx

```diff
@@ -27,15 +27,11 @@ export function PanelLinks({ panelLinks, onShowPanelLinks }: Props) {
   if (panelLinks.length === 1) {
     const linkModel = onShowPanelLinks()[0];
     return (
-      <a
-        href={linkModel.href}
-        onClick={linkModel.onClick}
-        target={linkModel.target}
-        title={linkModel.title}
-        className={styles.singleLink}
-      >
-        <Icon name="external-link-alt" size="lg" />
-      </a>
+      <PanelChrome.TitleItem>
+        <a href={linkModel.href} onClick={linkModel.onClick} target={linkModel.target} title={linkModel.title}>
```

Comment:
this does not really work well, the TitleItem has padding and hover effect, and should be the anchor / click target. 

Would add onClick and href to TitleItem so that it can handle rendering itself as an anchor or button depending on if it get's a href and onClick

Reply:
Thank you for the feedback, Torkel, I was hesitant on implementing it in that way for the sake of keeping it simple but I will follow this advice :).